### DISCUSSION TITLE
Installing on OpenBSD

### DISCUSSION THREAD

**Discussion by joji-github** [danielhrisca/asammdf#1006] [Q&A]
Hello.

I'm trying to install asammdf on OpenBSD 7.4. The build is failing. Any ideas/suggestions on what could be the problem.

Build transcript is attached.

[asammdf.log](https://github.com/danielhrisca/asammdf/files/14877059/asammdf.log)

I do have py3-numpy-1.24.1 installed as a package. But, the install process appears to rebuild numpy from source and that is where the failure appears to be happening.

Thank you.
joseph

**Comment by danielhrisca** [comment]
I don't know why it is trying to install numpy 2.0.0rc1, please try again to install numpy 1.26.4 before installing asammdf
